openedx/features/badging/helpers.py

```diff
@@ -108,7 +112,8 @@ def filter_earned_badge_by_joined_team(user, course, earned_badges):
         flag: Has user joined any team
         earned_badges: All badges earned in a course, specific to joined team
     """
-    course_team = CourseTeam.objects.filter(course_id=course.id, users=user).values('team__room_id').first()
+    # course_team = CourseTeam.objects.filter(course_id=course.id, users=user).values('team__room_id').first()
```

Comment:
Let's remove this commented line of code

Reply:
done